autodistill/detection/detection_base_model.py

```diff
@@ -31,10 +32,10 @@ class DetectionBaseModel(BaseModel):
     ontology: DetectionOntology
 
     @abstractmethod
-    def predict(self, input: str) -> sv.Detections:
+    def predict(self, input: str | np.ndarray | Image.Image) -> sv.Detections:
```

Comment:
NB: The `|` operator for specifying multiple type hints was introduced in Python 3.10. Since Python 3.10 no longer receives active support, I'll merge this. Thought I'd document here anyway!